src/Our.Umbraco.Vorto/Web/PropertyEditors/VortoPropertyEditor.cs

```diff
@@ -73,7 +73,10 @@ internal class VortoPreValueEditor : PreValueEditor
 
             [PreValueField("hideLabel", "Hide Label", "boolean", Description = "Hide the Umbraco property title and description, making the Vorto span the entire page width")]
             public bool HideLabel { get; set; }
-		}
```

Comment:
I prefer HideLabel to be the last prevalue option so move below the show filled languages option.